src/Sulu/Bundle/AdminBundle/Resources/js/components/CircleSelection/types.js

```diff
@@ -5,6 +5,6 @@ export interface Normalizer {
 
 export type SelectionData = {
     left: number,
-    radius?: number,
+    radius: number,
```

Comment:
Didn't we say that value should be able to not exist in case we are creating a point? Why is it a mandatory field now?

Reply:
Because of this conversation https://github.com/sulu/sulu/pull/5434#discussion_r496740460, I decided to make the radius mandatory. For a point it will be 0

Reply:
Would have preferred to fix it the other way round (that a point really does not have a radius), but I can see that it is easier this way.